src/BL/Commands/MaxTemprWeatherCommand.cs

```diff
@@ -11,10 +11,13 @@
 
 namespace BL.Commands
 {
-    public class MaxTemprWeatherCommand: IRequest<Unit>
+    //public class MaxTemprWeatherCommand: IRequest<Unit>
+    public class MaxTemprWeatherCommand : IRequest<double?>
     {
+        public string Cities { get; set; }
```

Comment:
Should be an array. Nobody designs Restful API in a way that input with collection is in a form of comma-delimited string.